integ-test/src/test/resources/json_test.json

```diff
@@ -1,14 +1,24 @@
 {"index":{"_id":"0"}}
-{"test_name":"json nested object", "json_string":"{\"a\":\"1\",\"b\":{\"c\":\"2\",\"d\":\"3\"}}"}
+{"test_name":"json nested object", "json_string":"{\"a\":\"1\", \"b\": {\"c\": \"3\"}, \"d\": [1, 2, 3]}"}
```

Comment:
can u also add a use case for inner json obj that has an array ?
`{"a":{"b":{"c":[....]}}}`

Reply:
"d" is mapped to an array.  Do you want an object within in an array?